pa11y/mappings/common/createCommonMappings.ts

```diff
@@ -0,0 +1,90 @@
+import { Options } from "wiremock-rest-client/dist/model/options.model";
+import wiremock from "../../lib/general";
+
+export default (baseUri?: string, options?: Options) => {
+  // Mapping for covers.
+  wiremock(baseUri, options).mappings.createMapping({
+    request: {
+      method: "GET",
+      urlPattern: "/api/v2/covers.*",
+    },
+    response: {
+      jsonBody: [
+        {
+          id: "870970-basis:134693959",
+          type: "pid",
+          imageUrls: {
+            small: {
+              url: "https://picsum.photos/98/160",
+              format: "jpeg",
+              size: "small",
+            },
+            large: {
+              url: "https://picsum.photos/142/172",
+              format: "jpeg",
+              size: "large",
+            },
+          },
+        },
+      ],
+    },
+  });
+
+  // Mapping for material list.
+  wiremock(baseUri, options).mappings.createMapping({
+    request: {
+      method: "HEAD",
+      urlPattern: "/list/default/.*",
+    },
+    response: {
+      jsonBody: {},
+    },
+  });
+
+  // Mapping for tokens (anonymous session).
+  wiremock(baseUri, options).mappings.createMapping({
+    request: {
+      method: "GET",
+      urlPath: "/dpl-react/user-tokens",
+    },
+    response: {
+      headers: {
+        "Content-Type": "application/txt",
+      },
+      body: 'window.dplReact = window.dplReact || {};\nwindow.dplReact.setToken("library", "fcd5c29a171f97b626d71eceffe1313f00a284b0")',
+    },
+  });
```

Comment:
Consider if this is still relevant and/or works.

We have previously specifically ignored requests to the CMS going through Wiremock. If we really want to ensure specific tokens are set we might need to mock the responses from Adgangsplatformen to the CMS instead.